spokestack/nlu/parsers/integer.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module contains the logic to parse integers from NLU results
+"""
+from typing import Any, Dict, Union
+
+from spokestack.nlu.parsers import maps
+from spokestack.nlu.parsers.digits import DIGIT_SPLIT_RE
+
+
+def parse(metadata: Dict[str, Any], raw_value: str) -> Union[int, None]:
+    """ Parser for Integer slots
```

Comment:
```suggestion
    """ Integer Parser
```
^ just changed this to match the others, though I think I might write them all as, e.g., "integer slot parser"